server/jsonutils/path.go

```diff
@@ -117,3 +117,74 @@ func FormatPrefixSuffix(key string) string {
 	}
 	return key
 }
+
+type JSONPathes struct {
+	pathes map[string]*JSONPath
+}
+
+func NewJSONPathes(pathes []string) *JSONPathes {
+	container := make(map[string]*JSONPath)
+
+	for _, path := range pathes {
+		container[path] = NewJSONPath(path)
+	}
+
+	return &JSONPathes{
+		pathes: container,
+	}
+}
+
+func (jpa *JSONPathes) String() string {
+	result := ""
+
+	for key := range jpa.pathes {
+		if result != "" {
+			result += ", "
+		}
+		result += key
+	}
+
+	return "[" + result + "]"
+}
+
+func (jpa *JSONPathes) Get(object map[string]interface{}) (map[string]interface{}, bool) {
```

Comment:
Please add comments about returning values (it's not clear now).

Reply:
Comment was added